beir.md

```diff
@@ -0,0 +1,224 @@
+# Reproducing Vespa ranking runs on the BEIR datasets 
+
+This document demonstrates how to reproduce zero-shot ranking runs on the BEIR benchmark using Vespa.
+
+Go through [experimenting yourself guide](experiment-yourself.md) first on how to build and deploy the app.
+
+In between the dataset, one can run the following to delete the index:
```

Comment:
not sure I understand

Reply:
What part do you not understand?

Reply:
in between the dataset

I think you mean "do this to remove an index"?